src/crashlytics/paramInterface.ts

```diff
@@ -0,0 +1,26 @@
+export interface FirebaseFilterInterval {
+    startTime: string;
+    endTime: string;
+}
+
+export interface PageDetails {
+    pageSize: string
+    pageToken: string
+}
+
+export interface FirebaseFilters {
+    categories: Array<string>;
+    customKeys: Array<string>;
+    eventType: Array<string>;
+    hasCrashlyticsInsights: Boolean;
+    manufacturerModels: Array<string>;
+    osVersions: Array<string>;
+    rollouts: Array<string>;
```

Comment:
I think overall in the codebase we've preferred `string[]` to `Array<string>`.

Reply:
Thanks for catching it. Updated.